### DISCUSSION TITLE
Hide root

### DISCUSSION THREAD

**Discussion by naser1991** [rodit/SnapMod#182] [Q&A/Support]
Best way to install snapmod without chance to get banned!

**Comment by rodit** [comment]
Nobody has been banned for using SnapMod yet. There is no precaution for avoiding getting banned.

I have been using SnapMod since the beginning of its life (a while ago) with magisk and LSPosed (Riru) on Android 10 with no issues, warnings or bans.

**Comment by ghost** [comment]
you cannot hide root without hooking thing that are native. snapchat has high security for that.
